partcad/src/partcad/sketch_factory_enrich.py

```
@@ -19,45 +19,62 @@
 
-            pc_logging.debug("Initializing an enrich to %s" % self.source)
+            pc_logging.debug(f"Initializing an enrich to {source_project_name}:{source_sketch_name}")
+
+            super().__init__(ctx, source_project, target_project, config)
+            self.source_project = source_project
+            self.source_project_name = source_project_name
+
+            if ";" in source_sketch_name:
+                self.source_sketch_name = source_sketch_name.split(";")[0]
+                suffix = source_sketch_name.split(";")[1]
+                self.extra_with = list(map(lambda p: p.split("="), suffix.split(",")))
```

Ruff (0.8.2)

Unnecessary `map()` usage (rewrite using a list comprehension)

Replace `map()` with a list comprehension

(C417)